nav2_common/nav2_common/launch/has_node_params.py

```diff
@@ -0,0 +1,60 @@
+# Copyright (c) 2019 Intel Corporation
```

Comment:
I didn't realize you worked for intel?

Reply:
Fixing, sorry since it started as a blatant copy of `rewritten_yaml.py` I never know at which point it's "our" copyright.